src/main/java/com/prgrms/needit/domain/center/service/CenterService.java

```diff
@@ -0,0 +1,72 @@
+package com.prgrms.needit.domain.center.service;
+
+import com.prgrms.needit.common.error.ErrorCode;
+import com.prgrms.needit.common.error.exception.NotFoundCenterException;
+import com.prgrms.needit.domain.center.dto.CenterCreateRequest;
+import com.prgrms.needit.domain.center.dto.CenterDetailResponse;
+import com.prgrms.needit.domain.center.dto.CenterUpdateRequest;
+import com.prgrms.needit.domain.center.entity.Center;
+import com.prgrms.needit.domain.center.repository.CenterRepository;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class CenterService {
+
+	private final CenterRepository centerRepository;
+	private final PasswordEncoder passwordEncoder;
+
+	public CenterService(
+		CenterRepository centerRepository,
+		PasswordEncoder passwordEncoder
+	) {
+		this.centerRepository = centerRepository;
+		this.passwordEncoder = passwordEncoder;
+	}
+
+	@Transactional
+	public Long createCenter(CenterCreateRequest centerRequest) {
+		return centerRepository
+			.save(centerRequest.toEntity(passwordEncoder.encode(centerRequest.getPassword())))
+			.getId();
+	}
+
+	@Transactional(readOnly = true)
+	public CenterDetailResponse getCenter(Long centerId) {
+		return centerRepository
+			.findById(centerId)
+			.map(CenterDetailResponse::new)
+			.orElseThrow(
+				() -> new NotFoundCenterException(ErrorCode.NOT_FOUND_CENTER));
```

Comment:
```suggestion
		return new CenterDetailResponse(findActiveCenter(centerId));
```

findActiveCenter 메서드와 같은 역할을 하니 코드를 위와 같이 수정할 수 있겠네용!

Reply:
앗 여기 있었네